frontends/web/src/components/sidebar/sidebar.tsx

```diff
@@ -54,6 +61,11 @@ interface SwipeAttributes {
     active?: boolean;
 }
 
+export const panelStore = new Store<SharedPanelProps>({
+  activeSidebar: false,
+  sidebarStatus: '',
+});
+
```

Comment:
The old store now only contains things for sidebar. We will also eventually move this into the context, but we first have to convert the components that are using this store into functional cmpts. :)